preprocessing/raw_data/nlu-benchmark/2017-06-custom-intent-engines/README.md

```diff
@@ -0,0 +1,65 @@
+# Natural Language Understanding benchmark
```

Comment:
There seem to be a lot of files included with the nlu-benchmark.  Are all of these used, or can we only include the files that are relevant?

Reply:
The `nlu-benchmark` is added as a third-party repo, which only creates a symlink to the original repository instead of having a copy in our own. In such case, it might not be possible to only include a subset of files.

Reply:
I see.  It might be helpful to list somewhere which files are used in your experiments.

Reply:
Added some description in the README file to mention what files we used.